cmd/probe.go

```diff
@@ -13,8 +13,8 @@ var probeInstallOptions probe.Options
 // probeCmd represents the get command
 var probeCmd = &cobra.Command{
 	Use:   "probe",
-	Short: "Display probe information",
-	Long:  `Display probe information`,
+	Short: "Checks for supported kubearmor features in the current environment",
+	Long:  `Checks for supported kubearmor features in the current environment`,
```

Comment:
> Hey @Essietom, Nice PR! Some suggestions inline.
> 
> But I have a bigger question in general. We are leveraging a lot of tabs spaces to align things, and the printing is not centralised and is dynamic, which can break and is a bit difficult to understand. Can we leverage a text template here? and store things in a Structure. Ref https://pkg.go.dev/text/template I am not sure if text templates support colors but still Can you once check it out. I believe it could simplify the code a lot. It might be a bit late to address this and if you think it would be a big change we can take it up in the future.

i totally agree with this, I thought of that too, leveraging the template will help, i will check the link you shared and update the code

Reply:
so I figured we could even use the table writer to format the output uniformly and this is what the new output looks like:
<img width="833" alt="Screenshot 2022-08-25 at 01 54 43" src="https://user-images.githubusercontent.com/10106690/186544298-92b1f525-af7f-4df3-a6d3-e5ac29c5cb25.png">

Reply:
kindly review and let me know what you think